src/shared/ml-thirdparty-simulator/index.ts

```diff
@@ -71,6 +78,12 @@ export class Simulator {
     const targetUrl = '/parties/' + type.toString() + '/' + id
     const payload = mockPutPartiesRequest(type, id)
 
+    if (this.opts.delay) {
+      // Delay operations to simulate network latency in real communication
+      // with Mojaloop.
+      await this.delay(1500)
```

Comment:
I see you have default delay defined on 2000 here: src/server/plugins/index.ts
Maybe it would be good to use it or write a comment on why it is lower or define simulator delay somewhere in global config?